zerver/tests/test_events.py

```diff
@@ -2607,6 +2608,23 @@ def do_upload() -> None:
         error = schema_checker('events[0]', events[0])
         self.assert_on_error(error)
 
+    def test_public_export_notify_admins(self) -> None:
+        self.user_profile.is_realm_admin = True
+        do_send_notify_admins_event(self.user_profile.realm, False, [self.user_profile])
+
+        schema_checker = self.check_events_dict([
+            ('type', equals('realm_exported')),
+            ('sender', equals(settings.NOTIFICATION_BOT)),
+            ('recipient_ids', check_list(check_int)),
+            ('to_s3', check_bool),
+        ])
+
+        events = self.do_test(
+            lambda: do_send_notify_admins_event(self.user_profile.realm, False, [self.user_profile]),
+            state_change_expected=False)
```

Comment:
I'm not entirely sure if `False` here is correct.

Reply:
docs/subsystems/events-system.mdyes, `state_change_expected` should be false here.  You can read more about this in `docs/subsystems/events-system.md` on ReadTheDocs.